frontend/src/metabase/nav/components/WhatsNewNotification/WhatsNewNotification.unit.spec.tsx

```diff
@@ -47,16 +49,25 @@ const setup = ({
   const versionMock = createMockVersion({ tag: currentVersion });
 
   const [latest, ...older] = versions;
-  const mockSettings = createMockSettingsState({
+  const mockedSettings = mockSettings({
```

Comment:
I'd suggest you inline the value at line 70 instead because the use of **mockedSomething** is very rare in our codebase (try a text search with `mocked` and you'll see).

Reply:
Ah, apparently plugins need to be setup *after* `mockSettings` has been called: https://github.com/metabase/metabase/blob/82cf4594ca8516f9d72aada296c25306f4b684a8/frontend/test/__support__/enterprise.js#L2-L3 (the test does indeed fail otherwise)
Seems I won't be able to inline it, I can take out the whole state though and just use the usual mock prefix that we use

Reply:
yeah that works. we also do `state = createMockState(...)` in many places